src/test/java/com/brite_erp/tests/functional_tests/discuss/Discuss.java

```diff
@@ -22,4 +22,18 @@ public void checkCreateChannel() throws Exception{
         Thread.sleep(5000);
         Assert.assertTrue(pages.createChannelPage().channelName.getText().equals("AutomationTest1"));
     }
+
+    @Test
+    public void checkDirectMessageFunctionality() throws Exception{
+        pages.chooseERP().rightERP.click();
+        pages.login().login(ConfigurationReader.getProperty("username"), ConfigurationReader.getProperty("password"));
```

Comment:
please change this line to pages.login(); only. We have updated our framework and this is the only way it will work.